site/js/ui.js

```diff
@@ -40,7 +40,7 @@ var UI = {
             Server.Automaton.create( nfaview.serialize(), function( id ) {
                 document.body.style.cursor = 'default';
                 var input = $( '#sharer input' )[ 0 ];
-                input.value = document.location.toString().split( '#v' )[0] + '#v' + id;
+                input.value = document.location.toString().split( '#' )[0] + '#v' + id;
                 $( '#sharer' ).show();
```

Comment:
@vasspilka Do you think maybe it's time to switch to more mainstream coding style avoiding spaces within parens etc.?